agave_app/GLView3D.cpp

```diff
@@ -311,8 +311,9 @@ GLView3D::OnSelectionChanged(SceneObject* so)
   // has the effect of re-creating the manipulator tool,
   // which will effectively call origins.update to get the new
   // selection into the tool
+  MANIPULATOR_MODE mode = m_manipulatorMode;
   setManipulatorMode(MANIPULATOR_MODE::NONE);
-  setManipulatorMode(m_manipulatorMode);
+  setManipulatorMode(mode);

```

Comment:
the changes in this file are actually fixing a systematic bug, where I wasn't actually saving the previous manipulation mode.